app.json

```diff
@@ -21,6 +21,9 @@
     },
     "SERVE_API_DOCS": {
       "value": "true"
+    },
+    "HEROKU_DISABLE_AUTH": {
+      "value": "true"
```

Comment:
might be better to set that as env var in Heroku, just in case we want to turn it off?

Reply:
Okay, I might change this to `"required": "true"`